src/components/Table/GridTable.tsx

```diff
@@ -232,11 +232,15 @@ export function GridTable<R extends Kinded, X extends Only<GridTableXss, X> = {}
       tableState.columns
         .filter((c) => tableState.visibleColumnIds.includes(c.id))
         .flatMap((c) =>
-          c.expandColumns && tableState.expandedColumnIds.includes(c.id) ? [c, ...c.expandColumns] : [c],
+          c.expandColumns && Array.isArray(c.expandColumns) && tableState.expandedColumnIds.includes(c.id)
+            ? [c, ...c.expandColumns]
+            : [c],
         ) as GridColumnWithId<R>[],
     [tableState],
   );
 
+  const expandedColumnIds: string[] = useComputed(() => tableState.expandedColumnIds, [tableState]);
```

Comment:
Suggestion: Super minor maybe put this line right before line 267, since afaict that's the 1st time where this value is needed?